actors/user/src/main/java/org/sunbird/user/actors/util/AuthenticationHelper.java

```diff
@@ -0,0 +1,169 @@
+    Response response =
+        cassandraOperation.getRecordsByCompositeKey(
+            usrDbInfo.getKeySpace(), JsonKey.USR_EXT_IDNT_TABLE, externalIdReq);
+
+    List<Map<String, Object>> userRecordList =
+        (List<Map<String, Object>>) response.get(JsonKey.RESPONSE);
+
+    if (CollectionUtils.isNotEmpty(userRecordList)) {
+      Map<String, Object> userExtIdRecord = userRecordList.get(0);
+      Response res =
+          cassandraOperation.getRecordById(
+              usrDbInfo.getKeySpace(),
+              usrDbInfo.getTableName(),
+              (String) userExtIdRecord.get(JsonKey.USER_ID));
+      if (CollectionUtils.isNotEmpty((List<Map<String, Object>>) res.get(JsonKey.RESPONSE))) {
+        // user exist
+        user = ((List<Map<String, Object>>) res.get(JsonKey.RESPONSE)).get(0);
+      }
+    }
+    return user;
+  }
+
+  public static String getEncryptedData(String value) {
+    try {
+      return encryptionService.encryptData(value);
+    } catch (Exception e) {
+      throw new ProjectCommonException(
+          ResponseCode.userDataEncryptionError.getErrorCode(),
+          ResponseCode.userDataEncryptionError.getErrorMessage(),
+          ResponseCode.SERVER_ERROR.getResponseCode());
+    }
+  }
+
+  public static Map<String, Object> getClientAccessTokenDetail(String clientId) {
+    Util.DbInfo clientDbInfo = Util.dbInfoMap.get(JsonKey.CLIENT_INFO_DB);
+    Map<String, Object> response = null;
+    Map<String, Object> propertyMap = new HashMap<>();
+    propertyMap.put(JsonKey.ID, clientId);
+    try {
+      Response clientResponse =
+          cassandraOperation.getRecordById(
+              clientDbInfo.getKeySpace(), clientDbInfo.getTableName(), clientId);
+      if (null != clientResponse && !clientResponse.getResult().isEmpty()) {
+        List<Map<String, Object>> dataList =
+            (List<Map<String, Object>>) clientResponse.getResult().get(JsonKey.RESPONSE);
+        response = dataList.get(0);
+      }
+    } catch (Exception e) {
+      ProjectLogger.log("Validating client token failed due to : ", e);
+    }
+    return response;
+  }
+
+  public static Map<String, Object> getUserDetail(String userId) {
+    Util.DbInfo userDbInfo = Util.dbInfoMap.get(JsonKey.USER_DB);
+    Map<String, Object> response = null;
+    try {
+      Response userResponse =
+          cassandraOperation.getRecordById(
+              userDbInfo.getKeySpace(), userDbInfo.getTableName(), userId);
+      if (null != userResponse && !userResponse.getResult().isEmpty()) {
+        List<Map<String, Object>> dataList =
+            (List<Map<String, Object>>) userResponse.getResult().get(JsonKey.RESPONSE);
+        response = dataList.get(0);
+      }
+    } catch (Exception e) {
+      ProjectLogger.log("fetching user for id " + userId + " failed due to : ", e);
+    }
+    return response;
+  }
+
+  public static Map<String, Object> getOrgDetail(String orgId) {
```

Comment:
Use OrgClient.getOrgById added by Rajat.

Reply:
Deleted not used.